index.html

```diff
@@ -1086,9 +1086,25 @@
 
 ## Principles for Vulnerable People
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Some people, or groups of people, are at greater risk from the asymmetries of power created by the
+concentration of information power ([[?Automating-Inequality]]). [=People=] who for whatever reason
+are in a more [=vulnerable=] position with respect to data [=processing=] should be more proactively
+protected by the [=user agent=] as well as by [=parties=].
```

Comment:
Given the caveat below that it's generally inappropriate to identify (or try to identify?) that a person is vulnerable, I'm not sure how we can say to proactively protect that specific group of people.

Maybe it's that "[=Actors=] on the web should not take advantage of any power they have over vulnerable people and should strive to protect any vulnerable people they happen to identify from other actors."? (This could be a Principle.) And then maybe something about actors needing to get their designs reviewed with user advocates and other experts to make sure they're not accidentally doing so?

Reply:
In addition to these safety mechanisms, process transparency and auditability seem relevant, not just regarding services for vulnerable people but also in other situations too where risks associated with data flows cannot be simply eliminated.

We might also be mindful about creating new obstacles to creators of services for vulnerable people. Assuming we ultimately want more and better services for vulnerable people, making such services extremely difficult to implement could have unintended consequences.